module/zfs/arc.c

```diff
@@ -6947,7 +6971,7 @@ arc_memory_throttle(uint64_t reserve, uint64_t txg)
 	 * continue to let page writes occur as quickly as possible.
 	 */
 	if (current_is_kswapd()) {
-		if (page_load > MAX(ptob(minfree), available_memory) / 4) {
+		if (page_load > MAX(arc_sys_free / 4, available_memory) / 4) {
```

Comment:
These two numbers seem very different. One is the number of free pages in bytes the other seems to be the number of 32 bit words? Is this really the comparison?

Reply:
All of these values are in fact in bytes and this PR doesn't make any real change here.  It only removes the `minfree` local and the needless `btop()` -> `ptob()` conversion.  This is logic was adapted from OpenZFS and is  intended to act as a throttle when there is significant memory pressure.

That said, according the codecov report this `page_load` block was never run during the ZTS.  I find that pretty suspicious so we should revisit this, bit it may be that we simply never put enough memory pressure on the system.  But let's address that in a different PR if needed so we can avoid making any functional changes in this one.